src/reducers/userDataReducer.js

```diff
@@ -49,6 +50,7 @@ function userDataReducer(state = initialState, action) {
         });
       return Object.assign({}, state, { photoName: newPhotoName });
     case SET_MOST_RECENT_PROGRAM:
+      setProgramCode(state.mostRecentProgram);
```

Comment:
is this so that save on change language works?

Reply:
Yes if I remember right? Taking out since Aseem and I got rid of the save on language to spare the server